### ISSUE TITLE
Kotlin extensions from oneOf group don't support arrays with not-nulls

### ISSUE THREAD

**Issue by denispo3** [realm/realm-java#5717]
#### Goal

> I want to pass `Array<Long>` as query parameter to `oneOf` function from Kotlin extensions

#### Actual Results

> `Array<Long>` can't be passed to `oneOf` function, because Kotlin extensions for this function can consume only arrays of nullable types (for example `Array<Long?>` in my case) while with old `in` function everything works as expected. Perhaps you should add `out` variance to generics in these extensions like this:

```kotlin

fun <T : RealmModel> RealmQuery<T>.oneOf(propertyName: String, value: Array<out Long?>): RealmQuery<T> {
        return this.`in`(propertyName, value)
}

```

#### Version of Realm and tooling
Realm version(s): 4.3.3

Android Studio version: 3.0.1

**Comment by cmelchior** [comment]
Yes, looks like a mistake on our part. We should that 👍

**Comment by cmelchior** [comment]
Fixed in #5723
